Sources/SmithyRetriesAPI/RetryErrorType.swift

```diff
@@ -22,15 +21,3 @@ public enum RetryErrorType: Equatable {
     /// Doesn’t count against any budgets.  This could be something like a 401 challenge in HTTP.
     case clientError
 }
-
-public extension RetryErrorType {
-
-    func toCRTType() -> AwsCommonRuntimeKit.RetryError {
-        switch self {
-        case .transient: return .transient
-        case .throttling: return .throttling
-        case .serverError: return .serverError
-        case .clientError: return .clientError
-        }
-    }
-}
```

Comment:
This CRT-dependent extension was split off to its own file in SmithyRetries (see above.)